draft-irtf-panrg-path-properties.md

```diff
@@ -181,7 +181,8 @@ Therefore, a new technology may implement an existing use case related to differ
 
 Nodes may be able to send flows via one (or a subset) out of multiple possible paths, and an entity may be able to influence the decision which path(s) to use.
 Path Selection may be feasible if there are several paths to the same destination (e.g., in case of a mobile device with two wireless interfaces, both providing a path), or if there are several destinations, and thus several paths, providing the same service (e.g., Application-Layer Traffic Optimization (ALTO) {{RFC5693}}, an application layer peer-to-peer protocol allowing endpoints a better-than-random peer selection).
-Entities may select their paths to fulfill a specific goal by specifying target properties for their selected paths, e.g., related to performance or security.
+Entities can achieve a specific goal by specifying target properties for their paths, e.g., related to performance or security.
+Such paths can be provided by the network or selected by the entity itself from all available paths.
```

Comment:
```suggestion
Entities can express their intent to achieve a specific goal by specifying target properties for their paths, e.g., related to performance or security.
Then, a path can be selected which best meets the target property, e.g., the entity can select this path from all available paths or the entity can express the target property to the network and rely on the network to select an appropriate path.
```

I agree that the main point here is that the target property describes an intent, and the implementation of the path selection is left open.
I rephrased this change to express that _somebody_ is going to select from the available paths (and not necessarily reconfigure a single existing path to meet a target), as this is the "Path selection" section.

Reply:
I like your changes. The only thing that I would change is use the plural instead of singular for path since path selection might also be used in a multipath setting. Also I shortened the second sentence a bit.
```suggestion
Entities can express their intent to achieve a specific goal by specifying target properties for their paths, e.g., related to performance or security.
Then, paths can be selected which best meet the target property, e.g., the entity can select these paths from all available paths or express the target property to the network and rely on the network to select appropriate paths.
```

Reply:
I would also use "target properties" instead of "target property" in the second sentence.

Reply:
Thanks to both of you! I'll apply the suggested changes.